### PR TITLE
bootstrap_r2

### PR DESCRIPTION
**Pull request by sannanphan**

### CODE REVIEW

**Comment by bolliger32** [comment]
I think the failure here is do to swapping around of the ED fig numbering. #218 should fix this. Once that is merged, we should merge in master to this PR and then it should pass checks.

File: codes/plotting/extended_data_fig5.do [hunk 1]
```diff
@@ -1,25 +1,34 @@
...
-legend(lab(1 "FRA") lab(2 "IRN") lab(3 "ITA") lab(4 "KOR") lab(5 "USA") rows(1) region(lstyle(none))) ///
+tw connect r2 L if grp == 1, mc(gold) lc(gold) || rspike max min L if grp == 1, lc(gold*.7) ///
+|| connect r2 L if grp == 2, mc(maroon) lc(maroon) || rspike max min L if grp == 2, lc(maroon*.7) ///
+|| connect r2 L if grp == 3, mc(ebblue) lc(ebblue) || rspike max min L if grp == 3, lc(ebblue*.7) ///
+|| connect r2 L if grp == 4, mc(green) lc(green) || rspike max min L if grp == 4, lc(green*.7) ///
+|| connect r2 L if grp == 5, mc(black) lc(black) || rspike max min L if grp == 5, lc(black*.7) ///
+legend(order(1 3 5 7 9) lab(1 "FRA") lab(3 "IRN") lab(5 "ITA") lab(7 "KOR") lab(9 "USA") rows(1) region(lstyle(none))) ///
 xtitle(# fixed lags) ytitle(R-squared, height(10))
 
+outsheet * using "results/source_data/ExtendedDataFigure5_b.csv", replace
```
Comment (bolliger32) [inline codes/plotting/extended_data_fig5.do:31]: Should we add this as a sheet to `ExtendedDataFigure5_lags.xlsx` rather than as a separate CSV?

Comment (sannanphan) [reply codes/plotting/extended_data_fig5.do:31]: this is used later to be added to the xlsx. I just moved this one to the subfolder "indiv" though

Comment (bolliger32) [reply codes/plotting/extended_data_fig5.do:31]: Sounds good! I don't see the move to `indiv` though

**Pull request merged**